app/controllers/change_requests_controller.rb

```diff
@@ -256,14 +255,19 @@ def persist_change(change_request)
     elsif change_request.is_a? AddressChangeRequest
       puts "AddressChangeRequest"
       address = Address.find(change_request.object_id)
-      
-      a = geocode_address address
-      unless a.nil?
-        field_change_hash["latitude"] = a.latitude
+      if address && field_change_hash["action"] == "remove"
+        address.delete
+      else
+        a = geocode_address address
+        unless a.nil?
+          field_change_hash["latitude"] = a.latitude
+          field_change_hash["longitude"] = a.longitude 
         field_change_hash["longitude"] = a.longitude 
-      end
+          field_change_hash["longitude"] = a.longitude 
+        end
```

Comment:
Something looks very wrong about this code, since the indentation doesn't match the actual nesting of expressions, and it seems like we're assigning to the lat/long twice in a row. Perhaps it was a strange git merge conflict? Could you fix this up?

Reply:
ya not sure how this happened; reverting!